modules/shared/nRF52840/part1_build.mk

```diff
@@ -49,6 +49,11 @@ ASFLAGS += -I$(COMMON_BUILD)/arm/startup
 ASFLAGS +=  -Wa,--defsym -Wa,SPARK_INIT_STARTUP=0
 ASFLAGS += -D__STACKSIZE__=$(MAIN_STACK_SIZE) -D__STACK_SIZE=$(MAIN_STACK_SIZE)
 
+# Tracker
+ifeq ($(PLATFORM_ID),26)
+USE_PRINTF_FLOAT ?= n
```

Comment:
Is this a breaking change for customer applications using printf?

Reply:
Yes, but I have a workaround for this.

Reply:
Can you elaborate on the workaround? I tried `Serial.printlnf("pi=%f", M_PI);` on a Tracker with 5.4.1 and I got the expected `pi=3.141593` but with this branch, I get `pi=`

Reply:
What is the workaround because Tracker Edge/Monitor Edge uses a lot of floats and doubles

Reply:
Mostly in the jsmn JSON writer but also logging of various sensor data and GPS

Reply:
Just to re-iterate: this is a temporary change and will be resolved (restored and some other stuff thrown away) before this PR is merged. This is failing a lot of our automated tests, so won't be unnoticed.